.circleci/config.yml

```
@@ -496,6 +501,16 @@ jobs:
           name: Verify locales
           command: yarn verify-locales --quiet
 
+  test-storybook:
+    executor: node-browsers-medium-plus
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
```

I see that there is an error in CI about storybook not being installed. This should have been handled by the install script normally, but I'm guessing it's installing browsers in a directory not cached between jobs.

I'd recommend adding the install step here:

```suggestion
      - run:
          name: Install Playwright browsers
          command: yarn dlx playwright install
      - run:
```